packages/jest-each/src/bind.js

```diff
@@ -42,6 +42,30 @@ export default (cb: Function, supportsDone: boolean = true) => (...args: any) =>
           throw error;
         });
       }
+
+      if (
+        tableArg.length === 1 &&
+        typeof tableArg[0] === 'string' &&
+        tableArg[0].trim() === ''
+      ) {
+        const error = new ErrorWithStack(
+          'Error: `.each` called with an empty tagged template string of table data.\n',
```

Comment:
I think it's called "tagged template literal"

Reply:
good spot!